.travis.yml

```diff
@@ -46,11 +47,38 @@ jobs:
             repo: open-policy-agent/gatekeeper
             branch: master
             tags: false
-        - provider: script
-          skip_cleanup: true
-          script: make travis-dev-release
-          on:
-            repo: open-policy-agent/gatekeeper
-            branch: master
-            tags: true
-
+    - stage: "build, release"
+      if: repo = open-policy-agent/gatekeeper AND type != pull_request AND tag IS present
```

Comment:
This will trigger a release on any tagged branch, regardless of if it's master or not

Reply:
From https://docs.travis-ci.com/user/environment-variables/ it doesn't seem like we can filter on branch when the build is triggered by a tag.
> Note that for tags, git does not store the branch from which a commit was tagged.

and 

> for builds triggered by a tag, this is the same as the name of the tag (TRAVIS_TAG)

Reply:
Correct. The option here would be to use the `deploy` rule with a script provider to work around this. Dunno why this feature asymmetry exists.